workflow-ui/core-android/src/main/java/com/squareup/workflow/ui/WorkflowViewStub.kt

```diff
@@ -160,6 +173,7 @@ class WorkflowViewStub @JvmOverloads constructor(
         .also { newView ->
           if (inflatedId != NO_ID) newView.id = inflatedId
           newView.visibility = visibility
+          newView.background = background
```

Comment:
Oops, you're always stomping the newView's background, even if the stub's background is unset.